src/selectors.rs

```diff
@@ -1,9 +1,112 @@
 use std::{collections::BTreeMap, str::FromStr};
 
-use minijinja::{value::Value, Environment};
+use minijinja::{
+    value::{Object, Value},
+    Environment,
+};
 use rattler_conda_types::{Platform, Version, VersionSpec};
 use serde_yaml::Value as YamlValue;
 
+#[derive(Debug)]
+struct Env;
+impl std::fmt::Display for Env {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str("Env")
+    }
+}
+
+impl Object for Env {
+    fn kind(&self) -> minijinja::value::ObjectKind<'_> {
+        minijinja::value::ObjectKind::Plain
+    }
+
+    fn call_method(
+        &self,
+        state: &minijinja::State,
+        name: &str,
+        args: &[Value],
+    ) -> Result<Value, minijinja::Error> {
+        let _state = state;
+        match name {
+            "get" => {
+                let mut args = args.iter();
```

Comment:
should we also error if there is more than one argument? Just so that people don't confuse it with get_default?